canary/Models/Record.cs

```diff
@@ -199,10 +199,19 @@ public static (Record record, List<Dictionary<string, string>> issues) CheckGet(
 
             if (e.Message != null)
             {
-                foreach (string er in e.Message.Split(";"))
+                Exception baseException = e.GetBaseException();
```

Comment:
Should line 200 be "if (e.GetBaseException().Message != null) ?

Possible that the outer exception has no message but the wrapped inner message does?

Reply:
Very good point I will go ahead and fix that.